src/ComparedQueryable/NativeQueryable/Error.cs

```diff
@@ -2,28 +2,31 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-namespace System.Linq
+using System;
+
+namespace ComparedQueryable.NativeQueryable
 {
     internal static class Error
     {
         internal static Exception ArgumentNull(string message) => new ArgumentNullException(message);
 
         internal static Exception ArgumentNotIEnumerableGeneric(string message) =>
-            new ArgumentException(Strings.ArgumentNotIEnumerableGeneric(message));
+            new ArgumentException(message);

```

Comment:
There were some shared messages that corefx had here that I didn't think necessarily needed to be built out for error tracking in this project.